credentials/retailCoupon/credential.json

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/retailCoupon-context-v1.json"
+  ],
+  "type": ["VerifiableCredential", "ClippedCouponCredential"],
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "name": "Big Retail Store",
+    "url": "https://example.com",
+    "image": "https://playground.chapi.io/examples/retailCoupon/logo.png"
```

Comment:
I think this example wouldn't work because of `image` and `url` are undefined properties (I think, I may have missed their definitions).

Reply:
Added 
```
    "image": {
      "@id": "https://schema.org/image"
    },
    "url": {
      "@id": "https://schema.org/url"
    },
```
at the top level of the context so it can apply on the VerifiableCredential or VerifiableCredential.issuer level.